src/databricks/labs/ucx/source_code/files.py

```diff
@@ -14,47 +19,45 @@
 logger = logging.getLogger(__name__)
 
 
-class SourceFile(SourceContainer):
+class LocalFile(SourceContainer):
 
     def __init__(self, path: str, source: str, language: Language):
         self._path = path
         self._original_code = source
         # using CellLanguage so we can reuse the facilities it provides
         self._language = CellLanguage.of_language(language)
```

Comment:
```suggestion
        self._language = CellLanguage.of_language(language)
        
        def is_notebook(self) -> bool:
           return 'Databricks notebook source' in self._original_code
```

we _may_ need it, at least in the stage of folder listing.

Reply:
that would be a bit late, see my comments re loading